content/install-ubuntu.md

```diff
@@ -55,137 +53,135 @@ Ubuntu may need a special option enabled to install and boot from Ubuntu until t
+![Login Screen 1](/images/install-ubuntu/ubuntu-24.04-login-screen-1.png)
 
-![Login Screen 2](/images/install-ubuntu/ubuntu-login-2.png)
+![Login Screen 2](/images/install-ubuntu/ubuntu-24.04-login-screen-2.png)
 
 Select your User from the login screen and enter your user passphrase.
 
-### Online Accounts
+### Ubuntu Pro
 
-![Online Accounts](/images/install-ubuntu/ubuntu-online-accounts.png)
+![Ubuntu Pro](/images/install-ubuntu/ubuntu-24.04_initial-setup-2.png)
 
-Ubuntu asks users if they would like to sign into online accounts after the first login. This can be done now, or later, in <u>Settings.</u>
+Canonical offers Ubuntu Pro for free on up to 5 machines but it is not required to use Ubuntu and it's features and does not change your experience.
 
 ### Telemetry
 
-![Telemetry](/images/install-ubuntu/ubuntu-21.04-telemetry.png)
+![Telemetry](/images/install-ubuntu/ubuntu-24.04_initial-setup-3.png)
 
 Canonical asks you if you would like to send some system information to them to improve Ubuntu. Opting in or out of this option will not affect your Ubuntu features or experience.
 
 ### Ready to Go
 
-![Ready](/images/install-ubuntu/ubuntu-ready-to-go.png)
+![Ready](/images/install-ubuntu/ubuntu-24.04_initial-setup-4.png)
 
 Congratulations, Ubuntu is now installed on your system!
 
 ### Available Updates
 
 Often, after a fresh install some packages will have been updated since the ISO file was created. If there are new updates available you'll see a dialogue box pop up like this:
 
-![Updates Available](/images/install-ubuntu/ubuntu-21.04-new-updates.png)
+![Updates Available](/images/install-ubuntu/ubuntu-updater.png)
 
 You can opt to install the updates later, or install them now by clicking **Install Now.**
 
-Installing the updates will produce this screen asking for an administrative user's password:
-
-![Authorization](/images/install-ubuntu/ubuntu-21.04-installing-updates.png)
-
-The system will display the progress of updates through this screen:
-
-![Updates in Progress](/images/install-ubuntu/ubuntu-21.04-updates-in-progress.png)
-
 And often will prompt for a restart once complete:
 
-![Updates Complete](/images/install-ubuntu/ubuntu-21.04-updates-finished.png)
+![Updates Complete](/images/install-ubuntu/ubuntu-updater-finished.png)
 
 ### Install System76 Driver
 
-Once you've reinstalled Ubuntu, you'll need to download and install the <u>System76 Driver</u>. Open the <u>Terminal</u> (search <u>Terminal</u> from the Ubuntu (<font-awesome-icon :icon="['fab', 'ubuntu']"></font-awesome-icon>) dash or press <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>T</kbd>), then enter the following commands:
+Once you've installed Ubuntu, you'll need to download and install the <u>System76 Driver</u>. Open the <u>Terminal</u> (search <u>Terminal</u> from the Ubuntu (<font-awesome-icon :icon="['fab', 'ubuntu']"></font-awesome-icon>) dash or press <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>T</kbd>), then enter the following commands:
```

Comment:
![screenshot-2024-05-01-19-57-34](https://github.com/system76/docs/assets/98765732/44ba36e3-321a-44cc-b803-f8d0ff12e8bf)

I'm not sure why, but the line breaks in this section are borked.